modules/header/react/lib/parts/Search.tsx

```diff
@@ -255,13 +264,23 @@ export class Search extends React.Component<SearchProps, SearchState> {
       cursor: 'pointer',
     };
 
-    const {onSearchSubmit, ...props} = this.props;
+    const {
+      themeColor,
+      placeholder,
+      value,
+      rightAlign,
+      collapse,
+      onSearchSubmit,
+      ...elemProps
+    } = this.props;
 
     return (
       <React.Fragment>
         <IconButton
+          aria-label="Search"
```

Comment:
This makes me wonder if a label should always be required for `IconButton`s

Reply:
This seems really reasonable to me, but is probably outside the scope of this PR. @anicholls should this be filed as an issue?

Reply:
Yeah, that sounds like a great idea! If you could file that separately, that would be great 👍